### PR TITLE
Install `Newtonsoft.Json.dll` not only for plugins

### LINKED ISSUE #9110: Missing Newtonsoft.Json.dll if installed with no plugins selected / Git Extensions (3.5.0.11713)
After clean installation of Git Extensions (3.5.0.11713) with no plugins selected. trying to open Settings leads to  an exception (see diagnostics).

## Steps to reproduce

- Make clean installation of Git Extensions
- Select no plugins during installation
- Open Git Extensions
- Go to Tools -> Settings

## Did this work in previous version of GitExtensions

I think so

## Environment

- Git Extensions 3.5.0.11713
- Build 0a5ef9ca6ce87754a510aa02c7a96d8b915714ac (Dirty)
- Git 2.31.1.windows.1
- Microsoft Windows NT 10.0.19043.0
- .NET Framework 4.8.4341.0
- DPI 240dpi (250% scaling)

## Diagnostics

---------------------------
Error
---------------------------

System.IO.FileNotFoundException: Could not load file or assembly 'Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. The system cannot find the file specified.

File name: 'Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'

   at GitCommands.Settings.Setting.SettingOf`1.<>c.<GetValue>b__20_0(String value)

   at GitCommands.SettingsCache.TryGetValue[T](String name, T defaultValue, Func`2 decode, T& value)

   at GitCommands.Settings.SettingsContainer`2.TryGetValue[T](String name, T defaultValue, Func`2 decode, T& value)

   at GitCommands.Settings.SettingsContainer`2.TryGetValue[T](String name, T defaultValue, Func`2 decode, T& value)

   at GitCommands.Settings.SettingsContainer`2.GetValue[T](String name, T defaultValue, Func`2 decode)

   at GitCommands.Settings.SettingsPath.GetValue[T](String name, T defaultValue, Func`2 decode)

   at GitCommands.Settings.Setting.SettingOf`1.GetValue(String name)

   at GitCommands.Settings.Setting.SettingOf`1.get_Value()

   at GitUI.CommandsDialogs.SettingsDialog.Pages.GitConfigSettingsPage.SettingsToPage()

   at GitUI.CommandsDialogs.SettingsDialog.SettingsPageBase.LoadSettings()

   at GitUI.CommandsDialogs.FormSettings.LoadSettings()

WRN: Assembly binding logging is turned OFF.

To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.

Note: There is some performance penalty associated with assembly bind failure logging.

To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

### PR DESCRIPTION
**Pull request by mstv**
Fixes #9110

## Proposed changes

- Always install `Newtonsoft.Json.dll` - not only for plugins
- Install it from and into the app directory

## Screenshots 

N/A

## Test methodology 

- install without plugins
- run GE

## Test environment(s) 

- Git Extensions 33.33.33
- Build 42c2639e71f08bc96844065efafbd4637edbc1ee
- Git 2.31.1.windows.1
- Microsoft Windows NT 10.0.19042.0
- .NET Framework 4.8.4341.0
- DPI 96dpi (no scaling)

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: Setup/Product.wxs [hunk 1]
```diff
@@ -1596,18 +1597,15 @@
         </Feature>
         <Feature Id="Bitbucket" Title="Atlassian Bitbucket Server integration" Level="1">
           <ComponentRef Id="Bitbucket.dll" />
-          <ComponentRef Id="Newtonsoft.Json.dll" />
```
Comment (gerhardol) [inline Setup/Product.wxs:1599]: Why must these references be removed?

**Review by gerhardol — COMMENTED** [review]
+1

File: Setup/Product.wxs (hunk 1 above)
Comment (mstv) [reply Setup/Product.wxs:1599]: The plugins load the DLLs from the app dir. Seems to work.

**Review by RussKie — APPROVED** [review]
Thank you

If it works :shipit:

**Pull request merged**